src/controllers/user.controller.js

```diff
@@ -0,0 +1,82 @@
+const userService = require('../services/user.service');
+
+const get = async (_req, res) => {
+  res.status(200).send(await userService.getAll());
+};
+
+const getById = async (req, res) => {
+  const { id } = req.params;
+
+  const user = await userService.getById(id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const create = async (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    return res.sendStatus(400);
+  }
+
+  const newUser = await userService.createUser(name);
+
+  res.status(201).send(newUser);
+};
+
+const remove = async (req, res) => {
+  const { id } = req.params;
+  const user = await userService.getById(id);
+
+  if (!id) {
+    res.sendStatus(200);
+
+    return;
+  }
```

Comment:
The check for `!id` is unnecessary here because `id` is extracted from `req.params`. If `id` is not present, it would typically result in a 404 error earlier in the request lifecycle. Consider removing this check or handling it in middleware.